src/NLog/Targets/Wrappers/AsyncTargetWrapper.cs

```diff
@@ -125,7 +125,7 @@ public AsyncTargetWrapper(Target wrappedTarget, int queueLimit, AsyncTargetWrapp
 #else
             _requestQueue = new AsyncRequestQueue(10000, AsyncTargetWrapperOverflowAction.Discard);
 #endif
-            TimeToSleepBetweenBatches = 50;
+            TimeToSleepBetweenBatches = 0;
```

Comment:
Not sure if I like this change for NLog 4.5.3

Reply:
I can only say that it is a huge improvement everywhere I have applied it. But this PR is still work-in-progress, as I want the OverflowAction.Block to be without spin. When block occurs then it can take several seconds, and it should be without spin.

Reply:
I won't disagree on the improvement, but I think it's wise not to change defaults in patch releases. Any way, tag it as 4.6 for now.

Reply:
@304NotModified Reverted change of default value for TimeToSleepBetweenBatches, and removed the spin when OverflowAction.Block.

Reply:
If you are not comfortable, then I can also change the default value to not use ConcurrentRequestQueue for all platforms.